packages/flutter/test/cupertino/dialog_test.dart

```diff
@@ -329,8 +329,11 @@ void main() {
 
     // TODO(yjbanov): https://github.com/flutter/flutter/issues/99933
     //                A bug in the HTML renderer and/or Chrome 96+ causes a
-    //                discrepancy in the paragraph height.
-    const bool hasIssue99933 = kIsWeb && !bool.fromEnvironment('FLUTTER_WEB_USE_SKIA');
+    //                discrepancy in the paragraph height. However, Dart HHH
+    //                uses an older version of Chrome at this time.
+    const bool hasIssue99933 = kIsWeb &&
+        !bool.fromEnvironment('FLUTTER_WEB_USE_SKIA') &&
+        !bool.fromEnvironment('DART_HHH_BOT');
```

Comment:
I think this will prevent the HHH bot from being able to upgrade to Chrome 96+ because as soon as it does so it will at the same time need a different expectation.

Reply:
I believe it will turn it red again, but it will not block from upgrading.  Given that this bot has been red for 1.5 months, I'd much rather get it back into a working state and manage the failures when the roll takes place by reverting this change at that time. @athomas @whesse - does that plan sound OK to you?

Two alternatives come to mind:
* Check for the Chrome version explicitly, instead. Is that available to us here? (not sure if this is worth it, though)
* Simply skip this line of the test in Dart HHH, so that the bot remains green after the roll.

Reply:
This is fine with me but I do wonder if it would be easier to wait until the version of chrome on the HHH bot gets updated. @athomas @whesse It's not clear from our chat if that is going to be a difficult or time consuming task or not.

Reply:
Mainly I was waiting for the version to land in the SDK before updating 100, but the SDK CL showed some unexplained flakiness. I'll prepare the CL on Monday. Then you can decide when to land it.

Reply:
https://dart-review.googlesource.com/c/recipes/+/243502

Reply:
Thanks @athomas! I'll drop this PR then :).